projects/Buttons_and_ADC/README.md

```diff
@@ -0,0 +1,16 @@
+<!--
+    General guidelines
+    These are just guidelines, not strict rules - document however seems best.
+    A README for a firmware-only project (e.g. Babydriver, MU, bootloader, CAN explorer) should answer the following questions:
+        - What is it?
+        - What problem does it solve?
+        - How do I use it? (with usage examples / example commands, etc)
+        - How does it work? (architectural overview)
+    A README for a board project (powering a hardware board, e.g. power distribution, centre console, charger, BMS carrier) should answer the following questions:
+        - What is the purpose of the board?
+        - What are all the things that the firmware needs to do?
+        - How does it fit into the overall system?
+        - How does it work? (architectural overview, e.g. what each module's purpose is or how data flows through the firmware)
+-->
+# buttons_and_adc
```

Comment:
Looks like the folder name still shows up with capital letters. This is fine this time around since it is a bit of a pain to have to recreate the project and do all that over again. Just remember it for next time.